lib/puppet/provider/service/redhat.rb

```diff
@@ -39,6 +41,7 @@ def enabled?
   # Don't support them specifying runlevels; always use the runlevels
   # in the init scripts.
   def enable
+      chkconfig("--add", @resource[:name])
```

Comment:
The commit message says "Running chkconfig --add before chkconfig on fixes this issue, and
should be harmless on the unaffected OS versions." However, we're skipping the acceptance test on `rhel7`, so do we know that it really is harmless?

For example, a user could explicitly specify the `redhat` provider (as `systemd` is the default on `rhel7`): 

<pre>
service { '...':
  ...
  provider => redhat
}
</pre>

And since we're skipping the acceptance test on `rhel7` we won't know if we've regressed. But perhaps using the `redhat` provider on `rhel7` doesn't make any sense?